brave/browser-laptop
# Coverts DownloadItem and DownloadsBar into redux

Submitter Checklist:


Resolves #9260

Auditors: @bsclifton 

Test Plan:
- download an item
- check if download item is shown
- check if you can interact with it (note there is a problem with pause/resume, fixed here #9246)

Reviewer Checklist:

Tests
